README.md

```diff
@@ -69,28 +73,38 @@ optimization problem:
 Note that both of these formulations also include the budget constraint. To run the single-period 
 demo with the CQM risk bounding formulation, type:
 
-`python portfolio.py -m 'CQM' --max-risk 2500`
+`python portfolio.py -m 'CQM' --max-risk 1200`
 
 ```bash
 Single period portfolio optimization run...
 
+Loading data from provided CSV file...
+
 CQM run...
 
 Best feasible solution:
-AAPL	 72
-MSFT	  0
-AAL	  0
-WMT	  3
-
-Estimated returns: 17.24
-Purchase Cost: 983.81
-Variance: 2434.26
+AAPL	 31
+MSFT	  3
+AAL	  3
+WMT	 12
+
+Estimated Returns: 11.96
+Sales Revenue: 0.00
+Purchase Cost: 998.79
+Transaction Cost: 0.00
+Variance: 1196.06
 ```
 
 We can do similarly for the return-bounding formulation, with this command: 
 
 `python portfolio.py -m 'CQM' --min-return 17.5`
 
+##### CQM Transaction Cost Model 
+
+The demo allows the user to model transaction costs as a percentage of the total transactions value (sales and purchases). A default CQM run with a transaction cost factor of 10% can be done with the following command:
+
+`python portfolio.py -m 'CQM' -t 0.01` 
```

Comment:
Is it possible to add a bit about how this factors into the formulation/CQM?

Reply:
Yes. I added some more content there. Can you merge once approved? I don't have write permission. Thanks!

Reply:
Sure - we should fix that though. Not sure who has the ability to give you those permissions. Maybe @arcondello ?